app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -284,8 +292,29 @@ public static void settingsEvent(String key, String value) {
                 .queue();
     }
 
-    public static void settingsClickEvent(String key) {
+    public static void settingsEvent(Context context, String key, String value) {
```

Comment:
I think this one is tricky. Write some comment to explain we only want pref_key_turbo_mode in firebase?